src/domain/orchard_domain.rs

```diff
@@ -6,6 +6,7 @@ use core::fmt;
 use blake2b_simd::{Hash as Blake2bHash, State};
 use zcash_note_encryption_zsa::{note_bytes::NoteBytes, AEAD_TAG_SIZE};
 
+use crate::bundle::Authorized;
```

Comment:
You could put together imports by moving Authorized import at the same line as Authorization (line 18)

Reply:
Done.